### ISSUE TITLE
Table not scrolling

### ISSUE THREAD

**Issue by jehlers** [18F/atf-eregs#346]
Found this table: https://atf-eregs.18f.gov/478-125/2014-18392#478-125-f-2

It only scrolls when my mouse is on the scroll bar. Also looks like it is missing some data?

**Comment by brittag** [comment]
Hmm, I ran into this too, with confusingly inconsistent behavior: https://github.com/18F/atf-eregs/issues/304

**Comment by cmc333333** [comment]
I believe we can't scroll horizontally _in the header_. I believe the empty table is correct -- it's a format they're requesting, not providing actual data.

**Comment by brittag** [comment]
Yeah, on this table I can only horizontal scroll when hovering over the empty cells - I can't horizontal scroll when hovering over the table title or the table headers. This is sort of confusing to me as a user.

**Comment by jehlers** [comment]
Hmmm... interesting. We should probably do something to make this easier.

**Comment by jbarnicle** [comment]
The datatables.js plugin doesn't support scrolling in the header - oddly enough. There is a jquery hack that I have to make scrolling in the header work.  I'll create a pull request so that It can be evaluated.

**Comment by brittag** [comment]
For reference, https://atf-eregs.18f.gov/555-218/2015-25190 may be interesting for testing with - that table requires both horizontal and vertical scrolling to see all of the content.
